types/detect/detect.go

```diff
@@ -49,6 +76,6 @@ type SignalSourceComplete string
 //Finding is the main output of a signature. It represents a match result for the signature business logic
 type Finding struct {
 	Data        map[string]interface{}
-	Context     Event
+	Context     interface{}
```

Comment:
I think we should at least GDoc what we expect here. Event with envelope or just payload?

Reply:
I'm not sure either, we used to pass the payload here directly since we've sent the payload directly to the engine. I'm not sure what would be better...

Reply:
I've decided to keep it as the payload for now, i've renamed it as such and GDoc'd it.